react/src/Datasets/components/DatasetTable.tsx

```diff
@@ -114,6 +114,15 @@ export default function DatasetTable() {
         }
     }, [apiInfo]);
 
+    const RenderDatasetType = useCallback(
+        (rowData: Dataset) => (
+            <Tooltip title={apiInfo?.dataset_types[rowData.dataset_type].name || " "}>
+                <div>{rowData.dataset_type}</div>
```

Comment:
ditto, this is also slightly different from the above and maybe all of these could be a component with `dataset_type` as the single prop?